picard/script/parser.py

```diff
@@ -293,6 +312,12 @@ def parse_text(self, top):
                     text.append('\n')
                 elif ch == 't':
                     text.append('\t')
+                elif ch == 'u':
+                    hex = self.read_multi(4)
```

Comment:
Do you need to check that 4 characters were read? What about '\uabc'? Should this raise a unicode error or eof?